packages/crackql/PKGBUILD

```diff
@@ -0,0 +1,46 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=crackql
+_pkgname=CrackQL
+pkgver=53.f4d41cd
+pkgrel=1
+pkgdesc='GraphQL password brute-force and fuzzing utility'
+arch=('any')
+groups=('blackarch' 'blackarch-webapp' 'blackarch-exploitation' 'blackarch-fuzzer')
+url='https://github.com/nicholasaleks/CrackQL'
+license=('BSD 3-Clause License')
+depends=('python' 'python-virtualenv')
+makedepends=('git' 'python-pip')
+source=("git+https://github.com/nicholasaleks/$_pkgname.git")
+sha512sums=('SKIP')
+install="$pkgname.install"
+
+pkgver() {
+  cd $_pkgname
+
+  echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
```

Comment:
Use this https://github.com/BlackArch/blackarch-pkgbuilds/blob/4ae2d3f269ad76eda2d78f3ea172e9bba6fd1835/PKGBUILD-generic-git#L24 when there are tags available